fix: tutorial pages spelling and grammar (#3958)
dfinity/portal: docs/tutorials/developer-journey/level-2/2.4-intro-candid.mdx

dfx deploy
```

> Recall that the Motoko compiler will automatically generate the Candid service description file based off of this code. This Candid file can be found at `src/declarations/candid_example_backend/candid_example_backend.did` and will contain the following:

Reviewer: ```suggestion
Recall that the Motoko compiler will automatically generate the Candid service description file based on this code. This Candid file can be found at `src/declarations/candid_example_backend/candid_example_backend.did` and will contain the following:
```
